.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           go get -v -t -d ./...
       - name: Test
-        run: go test ./...
+        run: go test -race ./...
```

Comment:
More details [here](https://golang.org/doc/articles/race_detector.html).
We should always enable this flag in CI tests.